blog/content/post/2012-02-20-improve-productivity-by-tracking-work-environment-with-git-vim.md

```diff
@@ -1,14 +1,15 @@
 ---
-ID: 62695
-title: 'Improve productivity by tracking work environment with git - vim'
+title:   Improve productivity by tracking work environment with git - vim
+abstract: In this post I try to describe my vim configuration procedure and how it is
+connected with git. The whole concept is based on keeping all the configuration
+files in a separate directory. These files are connected using symbolic links in
+places where system or application requires it.
```

Comment:
text in the abstract should have at least one space (Hugo will report a problem if there is no indent), we are using indent like in the example in blog/content/post/YYYY-MM-DD-file. You can also look at my previous change here: https://github.com/3mdeb/news-and-ideas/pull/114/files